lib/macros_autocompletion/macro_completer.js

```diff
@@ -23,7 +23,26 @@ const provider = {
 
         const [prefix, macro_name_prefix, obrace] = prefix_match
 
-        const matching_macros = find_macros(lu.strip_comments(request.editor.getBuffer().getText()), escape_regex(macro_name_prefix))
+        var text = request.editor.getBuffer().getText() //By default we take macros from the current buffer
+        var match
+        var macrosFile
+        var texRootRex = /%(\s+)?!TEX macros(\s+)?=(\s+)?(.+)/g // Comment including the relative path to the macros file
+        var editor = atom.workspace.getActivePaneItem()
+        var file = editor.buffer.file
+        var path = file.path
+
+        while(match = texRootRex.exec(request.editor.getBuffer().getText())) {
+          macrosFile = absolute(path,match[4])
+        }
+        if (macrosFile != undefined) {
+          try {
+            text = fs.readFileSync(macrosFile, 'utf-8');
```

Comment:
Yeah, there is no reason to ignore them. Now it concatenates the ones from the current buffer with the ones in the specified file (if any).